python_src/src/lamp_py/postgres/postgres_schema.py

```diff
@@ -39,13 +39,6 @@ class VehicleEvents(SqlBase):  # pylint: disable=too-few-public-methods
     vp_stop_timestamp = sa.Column(sa.Integer, nullable=True)
     tu_stop_timestamp = sa.Column(sa.Integer, nullable=True)
 
-    # forign key to static schedule expected values
-    fk_static_timestamp = sa.Column(
-        sa.Integer,
-        sa.ForeignKey("static_feed_info.timestamp"),
-        nullable=False,
-    )
-
```

Comment:
i guess this is just associated with the trip now, rather than any of the specific events?

Reply:
Yes, our new logic that matches static schedules to realtime events makes it so all events for each service date should match to the same static schedule. So there's no way that different stops in the same trip would be matched to a different static schedule. Tracking this in the trips table should be totally sufficient. 

I really should of taken this out with the other PR that dropped the trip info from the events table, but I missed it.